src/System.IO.Compression.Brotli/System.IO.Compression.Brotli.csproj

```diff
@@ -12,24 +15,54 @@
     <OutputTypeEx>library</OutputTypeEx>
     <StartupObject />
   </PropertyGroup>
-  <!-- Platform specific properties -->
-  <PropertyGroup Condition="'$(Platform)' == 'amd64'">
+  <PropertyGroup Condition=" '$(Platform)' == 'x64'">
```

Comment:
In this case we should duplicate some removes, like write:
`<Compile Remove="System\Interop\Library.Unix.cs" />` in windows When and in OSX When both